index-pl.html

```diff
@@ -420,40 +420,40 @@ <h2>Maskology</h2>
   <div class="container">
 
     <p>
-      What happens to disease transmission if 60% of people wear a 60% effective mask?
-      Or 90% wear a 50% effective mask?
-      Or 50% wear a 90% effective mask?
-      This interactive lets you answer these questions.
+      Jak zmieniłoby się roznoszenie choroby gdyby 60% osób nosiło 60% skuteczne maski?
```

Comment:
Myślę, że jest ok, żeby zostawić "60% skuteczne maski", ale można też użyć "maski o skuteczności 60%" (tak jest użyte gdzieś w już przetłumaczonym fragmencie).

Reply:
Super, dzięki, jakoś mi to brzmiało koślawo.